packages/web/src/pages/SignIn/SignIn.tsx

```diff
@@ -19,10 +20,18 @@ export const SignIn = () => {
   return (
     <Container>
       <Header />
-      <SignInForm isLoading={isLoading} error={error} onSubmit={handleSubmit} />
+      <Container className="signInConrollersContainer">
```

Comment:
Container component was supposed to be used on the page top level only. Just group components into div or extract them, no need to re-use the Component with modified styling.